server/plugin/command.go

```diff
@@ -656,15 +664,33 @@ func getAutocompleteData(config *Configuration) *model.AutocompleteData {
 	subscriptionsAdd.AddTextArgument("Owner/repo to subscribe to", "[owner/repo]", "")
 	subscriptionsAdd.AddTextArgument("Comma-delimited list of one or more of: issues, pulls, pulls_merged, pushes, creates, deletes, issue_creations, issue_comments, pull_reviews, label:\"<labelname>\". Defaults to pulls,issues,creates,deletes", "[features] (optional)", `/[^,-\s]+(,[^,-\s]+)*/`)
 	if config.GitHubOrg != "" {
-		flags := []model.AutocompleteListItem{{
-			HelpText: "Events triggered by organization members will not be delivered (the organization config should be set, otherwise this flag has not effect)",
-			Hint:     "(optional)",
-			Item:     "--exclude-org-member",
-		}}
-		subscriptionsAdd.AddStaticListArgument("Currently supports --exclude-org-member", false, flags)
-	}
-	subscriptions.AddCommand(subscriptionsAdd)
+		subscriptionsAdd.AddNamedStaticListArgument("exclude-org-member", "Events triggered by organization members will not be delivered (the organization config should be set, otherwise this flag has not effect)", false, []model.AutocompleteListItem{
+			{
+				Item:     "true",
+				HelpText: "Exclude posts from members of the configured organization",
+			},
+			{
+				Item:     "false",
+				HelpText: "Include posts from members of the configured organization",
+			},
+		})
+	}
+	subscriptionsAdd.AddNamedStaticListArgument("render-style", "Determine the rendering style of various notifications.", false, []model.AutocompleteListItem{
+		{
+			Item:     "default",
+			HelpText: "The default rendering style for all notifications (includes all information). Can be omitted.",
```

Comment:
I think "Can be omitted" is a little confusing because it sounds like the `value` of this flag can be ommitted, even though the flag parsing code will not allow for that.

```suggestion
			HelpText: "The default rendering style for all notifications (includes all information).",
```